rules/etcd-peer-tls-enabled/raw.rego

```diff
@@ -0,0 +1,50 @@
+package armo_builtins
+
+# Check if peer tls is enabled in etcd cluster
+deny[msga] {
+	obj = input[_]
+	is_etcd_pod(obj)
+
+	etcdPod := input[_]
+	result = invalid_flag(etcdPod.spec.containers[0].command)
+
+	msga := {
+		"alertMessage": "Etcd encryption for peer connection is not enabled.",
+		"alertScore": 7,
+		"packagename": "armo_builtins",
+		"failedPaths": result.failed_paths,
+		"fixPaths": result.fix_paths,
+		"alertObject": {"k8sApiObjects": [etcdPod]},
+	}
+}
+
+is_etcd_pod(obj) {
+	obj.apiVersion == "v1"
+	obj.kind == "Pod"
+	count(obj.spec.containers) == 1
+	endswith(split(obj.spec.containers[0].command[0], " ")[0], "etcd")
+}
+
+
+# Assume flag set only once
+invalid_flag(cmd) = result {
+	full_cmd = concat(" ", cmd)
+	wanted = [
+		["--peer-cert-file", "<path/to/tls-certificate-file.crt>"],
+		["--peer-key-file", "<path/to/tls-key-file.key>"],
+	]
+
+	fix_paths = [{
+		"path": sprintf("spec.containers[0].command[%d]", [count(cmd) + i]),
```

Comment:
I know it's my tweaked code, but I noticed a problem here. What if only the second one is missing? the the fix path would be `len(cmd) + 1`, but there will be no `cmd[len(cmd)]` element. I'm not sure this is a problem though

Reply:
you are right, but maybe let's figure it out later on. In my opinion having one empty line in remediation should not be a significant problem